<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2160</width>
    <height>3840</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>11</y>
     <width>2141</width>
     <height>421</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 180pt &quot;Gill Sans MT&quot;;
background-color: ; /* Vintage cream color */
background-color: rgba(224, 214, 176, 150);
border: 2px solid #5d4037; /* Dark brown border */</string>
   </property>
   <property name="text">
    <string>CODA</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="camera_frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>1651</width>
     <height>2711</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 5px solid #5d4037; /* Matches your retro brown */
background-color: black;    /* Looks like a screen when off */
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="rank_container">
   <property name="geometry">
    <rect>
     <x>1670</x>
     <y>440</y>
     <width>481</width>
     <height>2721</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>479</width>
      <height>2719</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
